db-migrations/src/main/resources/db/migration/V2__arena-data.sql

```diff
@@ -1,27 +1,21 @@
-CREATE TYPE arena_operation_type AS ENUM (
-    'INSERT',
-    'UPDATE',
-    'DELETE'
-    );
-
-CREATE TYPE arena_ingest_status AS ENUM (
-    'NEW',
-    'INGESTED',
-    'RETRY',
-    'FAILED'
-    );
-
 CREATE TABLE arena_data
 (
-    id                  SERIAL PRIMARY KEY   NOT NULL,
-    table_name          VARCHAR              NOT NULL,
-    operation_type      arena_operation_type NOT NULL,
-    operation_pos       BIGINT               NOT NULL,
-    operation_timestamp TIMESTAMP            NOT NULL,
-    ingest_status       arena_ingest_status  NOT NULL DEFAULT 'NEW',
+    id                  SERIAL PRIMARY KEY NOT NULL,
+    table_name          VARCHAR            NOT NULL,
+    operation_type      VARCHAR            NOT NULL CHECK ( operation_type IN ('INSERT', 'UPDATE', 'DELETE')),
```

Comment:
Virker som en litt kjip måte å komme seg rundt problematikken med å inserte enum[]

Reply:
Greit nok syns jeg. Men ping meg når du er klar for å merge så skal jeg gjøre klar databasen

Reply:
Ja, men jeg har ingen annen måte... Så for meg var det enten å gjøre det litt kjipt i Databasen eller litt kjipt i koden, og da valgte jeg kjipt i databasen :p